.env.example

```
@@ -11,4 +11,5 @@ NODE_ENV=development
 # Google OAuth Configuration
 GOOGLE_CLIENT_ID=your-google-client-id-here
 GOOGLE_CLIENT_SECRET=your-google-client-secret-here
-GOOGLE_CALLBACK_URL=http://localhost:3000/api/v1/auth/google/callback
\ No newline at end of file
+GOOGLE_CALLBACK_URL=http://localhost:3000/api/v1/auth/google/callback
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_CALLBACK_URL key should go before the GOOGLE_CLIENT_ID key

(UnorderedKey)